doc/Language/classtut.pod6

```diff
@@ -853,6 +910,43 @@ say $o.^parents.map({ $_.^name }).join(', ');
 Finally C<$o.^name> calls the C<name> method on the metaobject, which
 unsurprisingly returns the class name.
 
+Given an object C<$mp3> and the C<MP3TagData>class definition from the section
+L<Private methods|/language/classtut#Private_methods>, we can inquire about its
+public methods with C<.^methods>:
+
+=begin code :skip-test<compile time error>
+my $mp3 = MP3TagData.new(filename => 'football-head.mp3');
+say $mp3.^methods(:local);
+# OUTPUT: (TWEAK filename title artist album year comment genre track version
+# type Submethod+{is-hidden-from-backtrace}.new)
+=end code
+
+X<|^methods>
```

Comment:
OK, I see it's moved here. Would be interesting then to anchor it to a place that defined it, instead of using it in an example, maybe just adding here a brief explanation

Reply:
Do you mean an anchor to the definition of the example (i.e., `MP3TagData`) in particular? Because I used the same format as used [here](https://github.com/Raku/doc/pull/3778/files/a8c3f582daf8d49cf599a67ca0292ea2e9798cfa#diff-a5d9faf4ee78401ed8a8a680beef89e7cd5aa90126d15963f2bfea5584c2170eL802-L803), and linked directly to the section where it's declared.

Reply:
Well, to the first place where you explain what `^methods` are.

Reply:
>Well, to the first place where you explain what ^methods are.

I'm unclear about what you mean here. I'm myself not explaining what `^methods` does since it was already part of this tutorial and the anchor is already placed there. As for what they're, that's already part of the [documentation](https://github.com/Raku/doc/blob/master/doc/Type/Metamodel/MethodContainer.pod6#L31-L52).

Reply:
OK, just leave it here then. No problem.